src/AquaCalc5000/Parsers/ParsedData.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AquaCalc5000.Parsers
+{
+    public class ParsedData
+    {
+        public string LocationIdentifier { get; set; }
+        public DateTime StartDate { get; set; }
+        public int Transect { get; set; }
+        public string UserId { get; set; }
+
```

Comment:
I think we will also need to parse the meter calibration equations from the CSV file.

```
      METER TYPE  ,Price AA 1:1 ST2
 METER CONST. C1  ,2.2048
 METER CONST. C2  ,0.0178
 METER CONST. C3  ,2.2048
 METER CONST. C4  ,0.0178
 METER CONST. C5  ,0.0
```